lambdas/get-addresses/src/app.ts

```diff
@@ -15,7 +21,7 @@ export const lambdaHandler = async (event: APIGatewayProxyEvent): Promise<APIGat
             return response;
         }
 
-        const addressService = new AddressService(config.addressLookupStorageTableName, DynamoDbClient);
+        const addressService = new AddressService(configService.config.AddressLookupTableName, DynamoDbClient);
```

Comment:
Just wondering if we could move this outside the lambdaHandler to maybe line 9 for instance. Just lessons from the java lambda world where is it adviced to instantiate outside the handler code.

Reply:
we identified this in our run through. We decided to keep it like this for now as the overhead is tiny compared to Java, however we will come back to address it. 

moving the class initialisations out of the handler makes it a bit of a pain to test.